sdk/monitor/monitor-opentelemetry-exporter/swagger/README.md

```diff
@@ -24,6 +24,6 @@ source-code-folder-path: ./src/generated
 input-file: https://github.com/Azure/azure-rest-api-specs/blob/e47988f3ccaa90681f11cce59c25c638ff305692/specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15_Preview/swagger.json
 add-credentials: false
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210121.1"
+  "@autorest/typescript": "6.0.0-beta.8"
```

Comment:
If you would like to regenerate with the latest code gen, you do not have to specify any version by deleting this part.